### PR TITLE
Implemented user's recently played tracks request

### LINKED ISSUE #1: Login screen - Spotify Auth
### Task description

Create a simple login screen only with a button to allow [Spotify oAuth](https://developer.spotify.com/web-api/authorization-guide/) that gets the permission to `user-read-recently-played` token and redirects to the user's profile initial page.

### Things to do:

- [x]  Create the screen with the button to Spotify auth;
- [x]  Make the request to retrieve the token and store temporarily;

### PR DESCRIPTION
**Pull request by JRobsonJr**
Adds implementation of the request for the user's recently played tracks and multiple routes in order for the application to allow logging and exhibition the songs retrieved.

Resolves #1, #2

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,8 +9,10 @@
     "build": "webpack --display-error-details"
   },
   "dependencies": {
+    "axios": "^0.18.0",
     "babel-core": "^6.26.0",
     "babel-loader": "^7.1.4",
+    "babel-polyfill": "^6.26.0",
```
Comment (JoseRenan) [inline package.json]: For what is this dependency?

File: src/containers/App/index.js [hunk 2]
```diff
@@ -1,11 +1,20 @@
...
+import Auth from '../Auth';
 
 class App extends Component {
     render() {
         return (
-            <LoginPage />
+            <div>
+                <Switch>
+                    <Route exact path='/' component={LoginPage} />
+                    <Route path='/home' component={UserPage} />
```
Comment (JoseRenan) [inline src/containers/App/index.js]: Don't you think this route have to be `exact` too?

File: src/containers/Auth/index.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+    componentWillMount() {
+        console.log(this.props.match);
+        localStorage.setItem('token', this.props.match.params.token);
+        localStorage.setItem('refresh', this.props.match.params.refresh);
+        this.props.history.replace('/home');
+    }
+    
+    render() {
+        return (<div>This shouldn't be showing.</div>);
```
Comment (JoseRenan) [inline src/containers/Auth/index.js]: Hello, are u okay? KKKKKKKKKKKKK

File: src/components/LoginButton/index.js [hunk 4]
```diff
@@ -3,9 +3,9 @@ import React from 'react';
 import './style.css';
 
 const LoginButton = () => (
-    <button className="btn spotify-button">
+    <a href="http://localhost:5000/auth/login" className="btn spotify-button">
```
Comment (JoseRenan) [inline src/components/LoginButton/index.js]: Is not better pass this URL as parameter by props?

File: package.json (hunk 1 above)
Comment (JRobsonJr) [reply package.json]: Sorry, this was supposed to solve issue #3 but it didn't work. I'll remove it.

File: src/components/LoginButton/index.js (hunk 4 above)
Comment (JRobsonJr) [reply src/components/LoginButton/index.js]: Are you asking me?

File: src/containers/App/index.js (hunk 2 above)
Comment (JRobsonJr) [reply src/containers/App/index.js]: I agree with you.

File: src/containers/Auth/index.js (hunk 3 above)
Comment (JRobsonJr) [reply src/containers/Auth/index.js]: Yes!

File: src/components/LoginButton/index.js (hunk 4 above)
Comment (JoseRenan) [reply src/components/LoginButton/index.js]: KKKKKKKK yes

File: src/components/LoginButton/index.js [hunk 5]
```diff
@@ -1,11 +1,16 @@
...
-    <button className="btn spotify-button">
+const LoginButton = ({url}) => (
+    <a href={url} className="btn spotify-button">
         <i className="fa fa-spotify"/>
-    </button>
+    </a>
 );
 
+LoginButton.propTypes = {
+    url: PropTypes.string.isRequired
```
Comment (JoseRenan) [inline src/components/LoginButton/index.js]: don't you think urlToAuth is better?

File: src/containers/UserPage/index.js [hunk 6]
```diff
@@ -1,16 +1,24 @@
...
+        this.spotifyWebApi = new SpotifyWebApi();
         this.state = {
-            tracks: [] //TODO Request the user's recently played tracks
+            tracks: []
         }
     }
 
+    componentWillMount() {
+        this.spotifyWebApi.getRecentlyPlayedTracks().then(tracks => {
+            this.setState({...this.state, tracks: tracks.map(track => new Track(track.track))})
```
Comment (JoseRenan) [inline src/containers/UserPage/index.js]: i think `tracks.map(track => new Track(track.track))` will looks better if extracted to a const and change `tracks` to `results` and `track.track` to `result.track`

File: src/components/LoginButton/index.js (hunk 5 above)
Comment (JRobsonJr) [reply src/components/LoginButton/index.js]: Ok, my angel.

File: src/containers/UserPage/index.js (hunk 6 above)
Comment (JRobsonJr) [reply src/containers/UserPage/index.js]: Actually, I think it looks good right now u_u

**Pull request merged**